lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -86,8 +86,6 @@
 #define TEE_STORAGE_PRIVATE_REE	 0x80000000
 /* Storage is the Replay Protected Memory Block partition of an eMMC device */
 #define TEE_STORAGE_PRIVATE_RPMB 0x80000100
-/* Storage is provided by a SQLite database in the normal world filesystem */
-#define TEE_STORAGE_PRIVATE_SQL  0x80000200
```

Comment:
OK let's leave the burden on TA developers ;-) it's quite unlikely that many people are using SQL FS anyways (hence this PR to begin with).